### PR TITLE
feat: Add optional UMD style module export

### PR DESCRIPTION
**Pull request by dobromir-hristov**
Add ability to export the locales into an UMD styled export object. 
References issue #18

### CODE REVIEW

File: src/Generator.php [hunk 1]
```diff
@@ -40,8 +42,15 @@ public function generateFromPath($path)
             }
         }
 
-        return 'export default '
-            . json_encode($locales, JSON_PRETTY_PRINT | JSON_UNESCAPED_UNICODE) . PHP_EOL;
```
Comment (martinlindhe) [inline src/Generator.php:44]: The  PHP_EOL got lost, which is half of the cause of the failing tests

File: src/Generator.php [hunk 2]
```diff
@@ -155,4 +164,33 @@ private function removeExtension($filename)
...
+    }
+
+    /**
+     * Returns an ES6 style module
+     * @param string $body
+     * @return string
+     */
+    private function getES6Module($body)
+    {
+        return "export default {$body};";
```
Comment (martinlindhe) [inline src/Generator.php]: The trailing ; is the other half of the cause of the failing test

**Comment by martinlindhe** [comment]
Hi, this looks good! However, could you please fix the minor nitpicks i mentioned, in order for the tests to pass.

You can verify tests locally by running (from the root folder of this repository):

    vendor/phpunit/phpunit/phpunit

**Comment by dobromir-hristov** [comment]
Fixed those but its still not passing ☹️

**Comment by martinlindhe** [comment]
There's only some issue with hhvm, the "traditional" php versions are happy. So i'll merge this, thanks for your work!

**Comment by dobromir-hristov** [comment]
Uhuu 🎆

**Comment by martinlindhe** [comment]
I went ahead and disabled the hhvm test, as it has acted up before. Also, version 0.1.14 is released, including this PR, which you now can fetch using composer

**Pull request merged**